themes/bootstrap3/templates/Recommend/SideFacets/cluster-list.phtml

```diff
@@ -43,7 +43,20 @@
       <span class="text"><?=$this->transEsc('see all')?> ...</span>
     </a>
   <?php endif; ?>
-  <a class="facet narrow-toggle <?=$moreClass ?>" href="#" onclick="return lessFacets('narrowGroupHidden-<?=$this->escapeHtmlAttr($this->title) ?>')">
+  <a class="facet narrow-toggle <?=$moreClass ?> less-facets" href="#" data-title="<?=$this->escapeHtmlAttr($this->title) ?>">
     <span class="text"><?=$this->transEsc('less')?> ...</span>
   </a>
 <?php endif; ?>
+<?php
+$script = <<<JS
+$('.more-facets').click(function(e) {
+  title = $(this).data('title');
+  return moreFacets('narrowGroupHidden-' + title);
+});
+$('.less-facets').click(function(e) {
+  title = $(this).data('title');
+  return lessFacets('narrowGroupHidden-' + title);
+});
+JS;
+?>
+<?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, $script, 'SET');
```

Comment:
This is a much cleaner style than the inline `onclick`s, I thank you for that.

I'm concerned that adding event listeners at the end of every facet will duplicate listeners on earlier facets. Can we centralize this in one spot at the end of all the side facets?

Reply:
I will continue with that on #1639